next.config.js

```diff
@@ -221,7 +221,7 @@ module.exports = async (phase) => {
     },
     async redirects() {
       return [
-        { source: "/teachers", destination: "/", permanent: true },
+        { source: "/ai", destination: "/", permanent: false },
```

Comment:
@tweakster does this change (coupled with my follow-up on your previous comment) relieve your concern about the redirect caching?

Reply:
It was a question more than a concern, but if others are happy I am.